R/utils.R

```diff
@@ -223,22 +223,22 @@ file_copy <- function(..., overwrite = TRUE) {
   ok
 }
 
-sys_which <- function(name) {
-  path <- Sys.which(name)
-  if (!nzchar(path)) {
-    stop(sprintf("Did not find '%s'", name), call. = FALSE)
-  }
-  unname(path)
-}
-
-system3 <- function(command, args) {
-  res <- suppressWarnings(system2(command, args, stdout = TRUE, stderr = TRUE))
-  code <- attr(res, "status") %||% 0
-  attr(res, "status") <- NULL
-  list(success = code == 0,
-       code = code,
-       output = res)
-}
+#sys_which <- function(name) {
```

Comment:
I think we can remove these lines too if they're no longer needed.

Reply:
Done